test/package.json

```diff
@@ -59,7 +59,7 @@
     "codechain-primitives": "^1.0.4",
     "codechain-stakeholder-sdk": "https://github.com/junha1/codechain-stakeholder-sdk-js#master",
     "elliptic": "^6.4.1",
-    "foundry-rpc": "git://github.com/CodeChain-io/foundry-rpc-js.git#alpha-2",
+    "foundry-rpc": "git://github.com/somniumism/foundry-rpc-js.git#alpha-2",
```

Comment:
I pushed a [commit](https://github.com/somniumism/foundry-rpc-js/commit/3fb65ddb869eb8c998799d11a00485887d336bd6) onto alpha-2 branch in my [somniumism/foundry-rpc-js](https://github.com/somniumism/foundry-rpc-js/tree/alpha-2) repository to remove unnecessary score in the Block. And I modify the package.json in test.